recipes/mne-python_1.4/construct.yaml

```diff
@@ -134,6 +134,8 @@ specs:
   - plotly =5.14.1
   - vtk =9.2.6  # [arm64 or not osx]
   - vtk =9.2.5  # [osx and not arm64]
+  - git =2.40.1  # [win]
+  - make =4.3  # [win]
```

Comment:
These are just `[win]` because I think macOS and Windows people can be expected to have them on their own somehow (`brew` or `apt`). Windows people are unlikely unless they're using msysgit which is not super likely, and even if they are it probably/hopefully won't cause problems. (I installed these and was able to use them.)

Reply:
Yes this makes sense. On macOS you're likely to have the Xcode CLI tools installed, which ship git